.devcontainer/devcontainer.json

```diff
@@ -25,9 +25,8 @@
     2222
   ],
 
-  "onCreateCommand": "fisher install jorgebucaran/nvm.fish && nvm install",
```

Comment:
These commands more normally become available once the container is built&hellip;&hellip;

but for sure, were not leading to frequent successful builds
(even tho sometimes the errors were too slow to catch the builds
and ended up successful)